test/parallel/test-buffer-readint16be.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readInt16BE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readInt16BE(1);
+
+assert.strictEqual(result, 21672);
+
+assert.throws(
+  () => {
```

Comment:
Braces are always used in node code. I'm not sure if there's an explicit check when linting currently, but that's how we've been doing it.